src/templates/content-blocks/content_blocks.test.ts

```diff
@@ -19,28 +20,82 @@ describe('Content Blocks', () => {
       'Content-Type': 'application/json',
     },
   }
-  const response: ServerResponse = { message: 'success' }
-
-  it('calls templates.content_blocks.get()', async () => {
-    mockedRequest.mockResolvedValueOnce(response)
-    const contentBlockId = 'YOUR_CONTENT_BLOCK_ID'
-    expect(
-      await braze.templates.content_blocks.get({
-        content_block_id: contentBlockId,
-      }),
-    ).toBe(response)
-    expect(mockedRequest).toBeCalledWith(
-      `${apiUrl}/content_blocks/info?content_block_id=${contentBlockId}`,
-      undefined,
-      options,
-    )
-    expect(mockedRequest).toBeCalledTimes(1)
+
+  describe('calls templates.content_blocks.get()', () => {
+    const response: ContentBlockResponse = {
+      content: '',
+      content_block_id: '',
+      content_type: '',
+      created_at: '',
+      description: '',
+      inclusion_count: 0,
+      inclusion_data: [],
+      last_edited: '',
+      name: '',
+      tags: [],
+      message: 'success',
+    }
+
+    it('with required params', async () => {
```

Comment:
Nit: update test description here and below:

```suggestion
    it('is called with required params', async () => {
```

Reply:
updated